src/manifests.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+#
+# Learn more at: https://juju.is/docs/sdk
+
+import logging
+from typing import Dict
+
+from ops.manifests import ConfigRegistry, Manifests, Patch
+
+log = logging.getLogger(__name__)
+
+
+class SetCNIPath(Patch):
+    """Change CNI path to install SRIO-CNI plugin to"""
+
+    def __call__(self, obj) -> None:
+        obj.spec.template.spec.volumes[0].hostPath.path = self.manifests.config[
```

Comment:
While not strictly necessary, its probably good to add a check to make sure you are only applying this to the cnibin volume, and maybe add another check to make sure the necessary fields exist on the obj before accessing. If more resources are added to the manifest, or more volumes are added in the future and the volume order were to be different, we wouldnt want every volume to get that change.